app/routes/cms.logout.tsx

```diff
@@ -0,0 +1,18 @@
+import type { ActionFunction, LoaderFunction } from "remix";
+import { Box } from "@chakra-ui/react";
+import { redirect } from "remix";
+import { logout } from "~app/auth";
+
+export const action: ActionFunction = ({ request }) => {
+	return logout(request);
+};
+
+export const loader: LoaderFunction = () => {
+	return redirect("/");
+};
+
+export default function Logout() {
+	return <Box />;
+}
```

Comment:
This can probably be done better

Reply:
But how? One isn't allowed to export an empty element `<></>`.